### ISSUE TITLE
Research how to test React components the best.

### ISSUE THREAD

**Issue by hdavidzhu** [UrbanPod/PodTestPrototype#26]
- [ ] Look more into Mocha and Chai.
- [ ] See if a headless browser is appropriate.
  - [ ] If so, consider between Phantom and Casper (?)

**Comment by sihrc** [comment]
What do you mean by best? Most quickly? Easiest to implement? I'm a fan of simplest solution to perform basic integration tests.

**Comment by hdavidzhu** [comment]
Unit / functional tests.

**Comment by sihrc** [comment]
I don't think we have the time to spare for unit / functional tests. especially not on an initial prototype
